### PR TITLE
Add 25 Years of Java and JavaScript post

### PR DESCRIPTION
**Pull request by mraible**
This blog post has the following features:

- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been ran through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2020-12-04-25-years-javascript-java.adoc [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+image::{% asset_path 'blog/25-years-javascript-java/jquery.png' %}[alt=jQuery: write less, do more,width=400,align=center]
+
+In 2009, https://nodejs.dev/learn/a-brief-history-of-nodejs[Node.js was created], as well as the first version of `npm`. Node allows you to run JavaScript outside of a browser and JavaScript found itself well-positioned on the server.
+
+image::{% asset_path 'blog/25-years-javascript-java/node.png' %}[alt=Node.js: designed to build scalable network applications,align=center,width=400]
+
+In 2013, GitHub created Electron and introduced a way to write multi-platform desktop applications with JavaScript. Some of the most often-used software developers use today (Slack and VS Code) are powered by Electron!
+
+In 2014++, along came React, Angular (with TypeScript), and Vue. They continue to be the dominant frameworks for writing browser-based applications.
```
Comment (bdemers) [inline _source/_posts/2020-12-04-25-years-javascript-java.adoc:58]: `++` ?

File: _source/_posts/2020-12-04-25-years-javascript-java.adoc [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+Java didn't have much innovation while the JavaScript Renaissance was happening. I was a Java developer at LinkedIn in 2007-2008, then moved to full-time frontend development with GWT and jQuery at Evite in 2009.
+
+image:{% asset_path 'blog/25-years-javascript-java/spring-boot.png' %}[alt=Spring Boot,role="pull-right img-150px"]
+
+Java 8 was the next major thing to happen in JavaLand, and it wasn't released until March 18, 2014! A couple weeks later, Spring Boot 1.0 was released, on https://spring.io/blog/2014/04/01/spring-boot-1-0-ga-released[April Fools Day], no less!
+
+Spring Boot led to Spring Cloud and both helped fuel the microservices boom that continues today. MicroProfile was created to help with the stagnation in the Java EE world.
+
+Recently, there's been a lot of innovation happening with new frameworks like Micronaut and Quarkus. All the major server-side Java frameworks are _going native_ with GraalVM and making themselves relevant in a serverless, subatomic-startup-time, world.
```
Comment (bdemers) [inline _source/_posts/2020-12-04-25-years-javascript-java.adoc:94]: maybe add links for Micronaut and qurkaus?

**Review by bdemers — APPROVED** [review]
❤️

File: _source/_posts/2020-12-04-25-years-javascript-java.adoc (hunk 1 above)
Comment (mraible) [reply _source/_posts/2020-12-04-25-years-javascript-java.adoc:58]: I meant to imply "in 2014 and subsequent years". Maybe there's a better way to express this?

File: _source/_posts/2020-12-04-25-years-javascript-java.adoc (hunk 2 above)
Comment (mraible) [reply _source/_posts/2020-12-04-25-years-javascript-java.adoc]: Done.

**Pull request merged**